templates/commands/goldentest/new_test_cli.go

```diff
@@ -48,11 +48,13 @@ func (c *NewTestCommand) Desc() string {
 
 func (c *NewTestCommand) Help() string {
 	return `
-Usage: {{ COMMAND }} [options] <test_name>
+Usage: {{ COMMAND }} [options] <test_name> <location>
```

Comment:
I think the convention would be to add square brackets, like `[<location>]`, for things that are optional. We'd need to do this for `record` and `verify` too.

Reply:
done